backend/core/tests/test_add_github_collaborator_permission_check.py

```diff
@@ -0,0 +1,47 @@
+
+from django.test import TestCase
+
+from guardian.shortcuts import assign_perm
+
+from core.tests import factories as core_factories
+from core.models import Team
+from core.forms import AddGithubCollaboratorForm
+
+
+class TestAddGitHubCollaborator(TestCase):
+    def setUp(self) -> None:
+        self.staff_member = core_factories.UserFactory()
+        self.learner = core_factories.UserFactory()
+
+        self.staff_group = core_factories.TeamFactory()
+        self.staff_group.user_set.add(self.staff_member)
+
+        self.team_a = core_factories.TeamFactory()
+        self.team_a.user_set.add(self.learner)
+
+        self.team_b = core_factories.TeamFactory()
+
+        assign_perm(
+            Team.PERMISSION_MANAGE_CARDS,
+            self.staff_group,
+            self.team_a,
+        )
+        assign_perm(
+            Team.PERMISSION_MANAGE_CARDS,
+            self.staff_group,
+            self.team_b,
+        )
+
+        assign_perm(
+            Team.PERMISSION_MANAGE_CARDS,
```

Comment:
This doesn't change much, but should it not be `Team.PERMISSION_REVIEW_CARDS`. We are adding them as a collaborator/reviewer not a manager right?